geospaas_harvesting/ingesters.py

```diff
@@ -603,7 +603,9 @@ def _get_normalized_attributes(self, dataset_info, *args, **kwargs):
         " addresses or http addresses, not for ftp protocol")
 
         # Open file with Nansat
-        nansat_object = Nansat(nansat_filename(dataset_info), **nansat_options)
+        nansat_object = Nansat(nansat_filename(dataset_info),
+                               log_level=logging.getLogger("geospaas_harvesting").level,
```

Comment:
```suggestion
                               log_level=self.LOGGER.level,
```

Reply:
I have encountered that before, and now once again I debug that:
![image](https://user-images.githubusercontent.com/62345618/110484192-b8704480-80ea-11eb-8780-e0709e7d1dd0.png)

It does not work that way!

Reply:
done in 7b3a040